src/component-library/Slider/Slider.stories.tsx

```diff
@@ -0,0 +1,34 @@
+import { Meta, Story } from '@storybook/react';
+
+import { Slider, SliderProps } from '.';
+
+const Template: Story<SliderProps> = (args) => <Slider {...args} />;
+
+const Default = Template.bind({});
+Default.args = { label: 'Leverage', step: 1, minValue: 0, maxValue: 5, marks: false };
+
+const CustomMark = Template.bind({});
+CustomMark.args = {
+  label: 'Leverage',
+  step: 1,
+  minValue: 0,
+  maxValue: 5,
+  marks: true,
+  renderMarkText: (value) => `${value}x`
```

Comment:
This is a question more than anything—what does this look like when it renders? Wondering why we have the `x` at the end of the string.

Reply:
Nope ignore this :) realised this is only in the story